packages/rspeedy/plugin-react/src/entry.ts

```diff
@@ -105,6 +107,7 @@ export function applyEntry(
       const backgroundEntry = entryName
 
       mainThreadChunks.push(mainThreadName)
+      backgroundChunks.push(backgroundName)
```

Comment:
The `backgroundName` might include a hash such as `[contenthash:8]`. I'm uncertain if it will function correctly with `BannerPlugin.test`.

Reply:
Thanks for pointing this out, now I have changed to use the following regex (which is used for `RuntimeWrapperWebpackPlugin` now):

https://github.com/lynx-family/lynx-stack/blob/24291512b61cb2cba1838d9ba46384e8c99e265a/packages/rspeedy/plugin-react/src/entry.ts#L202